test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetSignTests.cs

```diff
@@ -34,7 +34,7 @@ public DotnetSignTests(MsbuildIntegrationTestFixture buildFixture, SignCommandTe
             _signFixture = signFixture;
         }
 
-        [Fact]
+        [PlatformFact(Platform.Windows, Platform.Linux)] // https://github.com/NuGet/Client.Engineering/issues/1076
```

Comment:
We should use public issues in Home, so any external contributor interested in helping out knows what the problem is that needs to be solved to get the test running again.

Reply:
Thanks! Changed the referenced issue.